cora_description/urdf/cora.urdf.xacro

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro" name="cora">
+
+  <link name="base_link">
+  </link>
+
+  <joint name="dummy_joint" type="fixed">
+    <parent link="base_link"/>
+    <child link="dummy_link"/>
+    <!--pose>0 0 0 1.57 0 1.57</pose-->
+  </joint>
+
+  <link name="dummy_link">
+    <visual>
+      <geometry>
+        <mesh filename="package://cora_description/models/base/meshes/base.dae"/>
+      </geometry>
+    </visual>
+
+    <collision name="collision">
+      <geometry>
+        <box size="3.58 3.53 12.21" />
+      </geometry>
+    </collision>
+
+    <!-- Numbers are from MeshLab -->
+    <inertial>
+      <pose>0 0.461760 0.262063</pose>
+      <mass value="50.964"/>
+      <inertia ixx="396.178589" ixy="0.0" ixz="0.0"
```

Comment:
Here's the inertia visualized. It might need a rotation maybe...

![cora_inertia](https://user-images.githubusercontent.com/1440739/94612204-0f076600-02a3-11eb-9b7d-a511c7b6f762.png)

Reply:
8e29019
Yep it does. I saw it when I was trying to fix the flotation, which... isn't fixed yet. But the inertia and collision looks more correct now:
![Screenshot from 2020-09-29 20-18-15](https://user-images.githubusercontent.com/7608908/94629599-49680780-0291-11eb-8027-80e333b4df98.png)
![Screenshot from 2020-09-29 20-18-09](https://user-images.githubusercontent.com/7608908/94629602-4b31cb00-0291-11eb-92fd-5f14150aa7db.png)
The inertia leans slightly downward on the stern, I think because there's a lot of mounting pieces in the back, and the cabin is hollow.